src/Admin/ReportAdmin.php

```diff
@@ -87,10 +88,19 @@ protected function configureDatagridFilters(DatagridMapper $datagridMapper)
                 'field_type' => TextType::class,
                 'callback' => function (ProxyQuery $qb, string $alias, string $field, array $value) {
                     if (!$value['value']) {
-                        return;
+                        return false;
+                    }
+
+                    // This is a kind of hack to know which class is selected if any
+                    if ($typeClass = $this->datagrid->getValues()['subjectType']['value']) {
+                        $ids = $this->reportRepository->findIdsByNameForClass($typeClass, $value['value']);
```

Comment:
Just tested again, this is not true here, because we do a sub request to get the ids to pass in the condition `IN`. This sub request has to be aware of the value of the other filter because it does not share the same builder.

Reply:
Ok tested it again and you're right! Fixed, thanks :)

Reply:
Awesome ! thanks ;-)